lib/querystring.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  StorageObject,
+  percentEscape,
+  unescapeBuffer
+} = require('internal/querystring');
 const QueryString = module.exports = {
   unescapeBuffer,
```

Comment:
`encode` and `decode` were added in https://github.com/nodejs/node/commit/4ce100fa622d97de51d7cccde2301653ee95bbe7 (v0.1.99). `unescapeBuffer` is slightly newer, but by not much, being added in https://github.com/nodejs/node/commit/6ff12c425e83522a72841f9966eefbe9a1eefc37 (v0.3.2). None of them have ever been documented.